components/EntriesList/EntriesList.js

```diff
@@ -26,37 +37,84 @@ export default function EntriesList() {
 
   return (
     <>
-      {reversedMoods.map((entry) => (
-        <Styled.Container key={entry._id}>
-          <Styled.AnimationContainer>
-            <div
-              style={{
-                backgroundColor: entry.color,
-                height: "100%",
-                width: "100%",
-              }}
-            ></div>
-            {/* <Animation color={entry.color} opacity={entry.intensity} /> */}
-          </Styled.AnimationContainer>
-          <Styled.Sentence>
-            <Styled.StaticText>You felt</Styled.StaticText> {entry.experience}.{" "}
-            <Styled.StaticText>More specifically</Styled.StaticText>{" "}
-            <Intensity value={entry.intensity} experience={entry.experience} />
-            <Styled.StaticText>
-              . You selected these tags:
-            </Styled.StaticText>{" "}
-            {entry.reactions.map((reaction, index, array) => (
-              <span key={index}>
-                {reaction}
-                {index < array.length - 1 && ", "}
-              </span>
-            ))}
-          </Styled.Sentence>
-          <Styled.Button onClick={() => handleDeleteEntry(entry._id)}>
-            delete mood
-          </Styled.Button>
-        </Styled.Container>
-      ))}
+      <Styled.Grid>
+        <AnimatePresence>
+          {reversedMoods.map((entry) => (
+            <motion.div
+              key={entry._id}
+              whileHover={{ scale: 1.05 }}
+              exit={{ opacity: 0, scale: 0.8, transition: { duration: 0.3 } }}
+            >
+              <Styled.Card onClick={() => router.push(`${entry._id}`)}>
+                <Styled.AnimationContainer>
+                  <div
+                    style={{
+                      backgroundColor: entry.color,
+                      height: "100%",
+                      width: "100%",
+                    }}
```

Comment:
you hopeI don't notice? 🤣😏

Reply:
style={{inlineStyling: `hidden from ${Merle}`}} // NOT WORKING!!!